google/cloud/storage/examples/storage_bucket_acl_samples.cc

```diff
@@ -246,10 +246,13 @@ void RunAll(std::vector<std::string> const& argv) {
   auto client = gcs::Client::CreateDefaultClient().value();
   std::cout << "\nCreating bucket to run the example (" << bucket_name << ")"
             << std::endl;
-  auto bucket_metadata = client
-                             .CreateBucketForProject(bucket_name, project_id,
-                                                     gcs::BucketMetadata{})
-                             .value();
+  (void)client
+      .CreateBucketForProject(bucket_name, project_id, gcs::BucketMetadata{})
+      .value();
+  // In GCS a single project cannot create or delete buckets more often than
+  // once every two seconds. We will pause for at least that long before
```

Comment:
s/for at least that long/until at least that time/ perhaps (et seq)?

Reply:
Done.